docs/design/autoware-interfaces/ad-api/features/index.md

```diff
@@ -0,0 +1,4 @@
+# Features of Autoware AD API
```

Comment:
[IMO] If you just list APIs, I feel this is better.
If you're planning to add more information in the future, as-is is okay.

```suggestion
# AD API list
```

Reply:
Same as https://github.com/autowarefoundation/autoware-documentation/pull/115#discussion_r894100598. There are other pages for the API list like [this](https://tier4.github.io/autoware-documentation/pr-7/design/autoware-interfaces/ad-api/list/).

Reply:
Mm, I think these pages can be merged. I feel splitting the pages doesn't help people understand.
@LalithVipulananthan What do you think about this?

Reply:
@kenji-miyake For example, is it as follows?

- Interface feature
    - /api/interface/version
- Driving feature
    - /api/driving/state
    - /api/driving/engage

Reply:
I'm not sure if I understand what you meant correctly, but I think it's yes.

Reply:
I agree with @kenji-miyake. Unless there are going to be a very large number of features, then it would be better to list them on the Features page with links to each individual feature page, rather than having a separate "List" page.

Reply:
@kenji-miyake @LalithVipulananthan 
I agree that if the menu is collapsed by default. There are already too many items and it is difficult to find. On my screen, I need to scroll to access "Configuration management" page.

Reply:
Fixed https://github.com/autowarefoundation/autoware-documentation/pull/115/commits/9f7d700d1af049783752ed9f2d31c58f0772db3a. But I could only find a way to expand or collapse all menu item. Do you know the way to partially expand the menu?

Reply:
Unfortunately, no. :pleading_face: